Error - A plugin in your app does not support the compulsory arm64 (64-bit support) architecture.

Hi, I'm trying to use this with Phonegap Build and cordova-plugin-admob-free.  However, the iOS build fails with the error "A plugin in your app does not support the compulsory arm64 (64-bit support) architecture.

Here is what it's saying the log:

ld: warning: ignoring file appName/Plugins/cordova-plugin-admob-mediation-adcolony/AdColony.framework/AdColony, file was built for unsupported file format ( 0x58 0x53 0x79 0x6D 0x0A 0x30 0x30 0x32 0x35 0x0A 0x66 0x31 0x36 0x31 0x35 0x31 ) which is not the architecture being linked (arm64): appName/Plugins/cordova-plugin-admob-mediation-adcolony/AdColony.framework/AdColony
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_AdColony", referenced from:
      objc-class-ref in libAdapterSDKAdColony.a(GADMAdapterAdColony.o)
      objc-class-ref in libAdapterSDKAdColony.a(GADMAdapterAdColonyInitializer.o)
  "_ADC_SET_USER_GENDER", referenced from:
      -[GADMAdapterAdColony addUserMetadata] in libAdapterSDKAdColony.a(GADMAdapterAdColony.o)
  "_ADC_USER_MALE", referenced from:
      -[GADMAdapterAdColony addUserMetadata] in libAdapterSDKAdColony.a(GADMAdapterAdColony.o)
  "_ADC_SET_USER_LATITUDE", referenced from:
      -[GADMAdapterAdColony addUserMetadata] in libAdapterSDKAdColony.a(GADMAdapterAdColony.o)
  "_ADC_USER_FEMALE", referenced from:
      -[GADMAdapterAdColony addUserMetadata] in libAdapterSDKAdColony.a(GADMAdapterAdColony.o)
  "_ADC_SET_USER_AGE", referenced from:
      -[GADMAdapterAdColony addUserMetadata] in libAdapterSDKAdColony.a(GADMAdapterAdColony.o)
  "_ADC_SET_USER_LONGITUDE", referenced from:
      -[GADMAdapterAdColony addUserMetadata] in libAdapterSDKAdColony.a(GADMAdapterAdColony.o)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** BUILD FAILED **

Any idea what could be causing that?

--- a/src/ios/AdColony.framework/AdColony
+++ b/src/ios/AdColony.framework/AdColony
@@ -1,5 +1,1 @@
-XSym
-0025
-f16151520e077b936c9f8a2996688c1e
 Versions/Current/AdColony
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      

--- a/src/ios/AdColony.framework/Headers
+++ b/src/ios/AdColony.framework/Headers
@@ -1,5 +1,1 @@
-XSym
-0024
-86c63de7bd8775780ac77380b5c049c4
 Versions/Current/Headers
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       

--- a/src/ios/AdColony.framework/Versions/Current
+++ b/src/ios/AdColony.framework/Versions/Current
@@ -1,5 +1,1 @@
-XSym
-0001
-7fc56270e7a70fa81a5935b72eacbe29
 A
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              

